packages/components/src/Notification/Notification.test.js

```diff
@@ -114,17 +114,24 @@ describe('Notification', () => {
 
 describe('NotificationContainer', () => {
 	describe('Timer Registry Utility', () => {
-		it('register, cancel, isRegistered', () => {
+		it('register, pause, resume, cancel, isRegistered', () => {
 			const notification = {
 				id: 'test-1',
 			};
-			const mockTimer = 1;
+			const mockTimer = {
+				pause: jest.fn(),
+				resume: jest.fn(),
+				cancel: jest.fn(),
+			};
 			const instance = new NotificationsContainer({ notifications: [] });
 			instance.registry.register(notification, mockTimer);
-			expect(instance.timerRegistry[notification.id]).toEqual(mockTimer);
 			expect(instance.registry.isRegistered(notification)).toEqual(true);
+			instance.registry.pause(notification);
```

Comment:
I won't block the PR for that because it was already like that and refactoring the tests would be out of scope, but we should not test internal functions like that anymore. That makes the test fragile